ArduPlane/radio.cpp

```diff
@@ -160,26 +139,29 @@ void Plane::rudder_arm_disarm_check()
 			// not at full right rudder
 			rudder_arm_timer = 0;
 		}
-	} else if ((arming_rudder == AP_Arming::RudderArming::ARMDISARM) && !is_flying()) {
-		// when armed and not flying, full left rudder starts disarming counter
-		if (channel_rudder->get_control_in() < -4000) {
-			uint32_t now = millis();
-
-			if (rudder_arm_timer == 0 ||
-				now - rudder_arm_timer < 3000) {
-				if (rudder_arm_timer == 0) {
-                    rudder_arm_timer = now;
-                }
-			} else {
-				//time to disarm!
-				arming.disarm(AP_Arming::Method::RUDDER);
-				rudder_arm_timer = 0;
-			}
-		} else {
-			// not at full left rudder
-			rudder_arm_timer = 0;
-		}
+        return;
 	}
+
+    // full left rudder starts disarming counter
```

Comment:
From a continuity perspective it seems unfortunate that this has a different look then the arming path, whereas before they were at least mirrored cases, which made it obvious when changing one how to change the other.

Reply:
Yes, I was going to revert it to make them look just the same again but discovered that was harder than it looked.

I was planning on making the first half look like the second half after this merge, just didn't want to complicate this patch.

I'll go back and arrange the second half to look like the first again - maybe rearrange it as it moves into RC_Channel

Reply:
This was done.